packages/eslint-config/index.js

```diff
@@ -0,0 +1,56 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+      legacyDecorators: true,
+    },
+  },
+  env: {
+    browser: true,
+    node: true,
+    es6: true,
+  },
+  globals: {
+    document: true,
+    window: true,
+    React: 'writable',
+  },
+  plugins: [
```

Comment:
Lets make the config smaller. We would not want to enforce a ton of stuff. Let the users extend to whatever they need

Reply:
@ankurkakroo2  do we need to provide very basic configuration? initially I thought to provide basic version but thought than what is intent of package ? Please confirm.

Reply:
We are already consuming airbnb and other rules which are massive in itself. I am not saying write as less as possible but rather, use only those configs which we are sure that the user should inherit.

The primary intention for these kind of packages is usually consistency among multiple teams.  We have learnt this before as well. The more we put, the harder it is for us.

Lets try to provide a just sufficient extensible config.

Reply:
Starting small is a good idea and carefully adding curated items to make it better